src/styles/routingTransitions.ts

```diff
@@ -0,0 +1,37 @@
+import { css } from '@emotion/core'
+import { transitions, sizes } from '@/shared/theme'
+import { TitleSection, VideoSection } from '@/views/ChannelView/ChannelView.style'
+import { InfoContainer } from '@/views/VideoView/VideoView.style'
+import { Container as HomeViewContainer } from '@/views/HomeView'
+import { Container as ChannelsViewContainer } from '@/views/VideosView/VideosView.style'
+
+export const routingTransitions = css`
+  .${transitions.names.fadeAndSlide}-enter {
+    opacity: 0;
+
+    & ${InfoContainer}, ${TitleSection}, ${VideoSection}, ${HomeViewContainer}, ${ChannelsViewContainer} {
```

Comment:
Do we need to manually specify all the containers here? This will be get hard to maintain very quickly